src/convertToObject.js

```diff
@@ -1,12 +1,68 @@
 'use strict';
 
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString === '') {
+    return {};
+  }
+
+  const firstObrabotka = sourceString;
+
+  const bigArr = firstObrabotka.split(';');
+
+  const arrFinal = bigArr.filter((str) => str.trim() !== '');
+
+  let smallArr;
+  const result = {};
+
+  for (let i = 0; i < arrFinal.length; i++) {
+    smallArr = arrFinal[i].split(':');
+
+    const key = smallArr[0].trim();
+    const value = smallArr[1].trim();
+
+    result[key] = value;
```

Comment:
Be cautious when assigning values to the 'result' object. If there are duplicate keys due to identical CSS properties, the last one will overwrite previous ones. This may or may not be the desired behavior depending on the use case.